server/src/main/java/com/coincoinche/websockets/messages/GameFinishedMessage.java

```diff
@@ -22,6 +34,21 @@ public String getMessageType() {
 
   @Override
   public String getJsonContent() {
-    return String.format("{\"win\":%s}", win);
+    String baseJsonContent =
+        String.format(
+            "\"win\":%s, \"yourTeamPoints\":%s, \"otherTeamPoints\":%s",
+            win, yourTeamPoints, otherTeamPoints);
+    StringBuffer eloUpdateJsonContent = new StringBuffer();
```

Comment:
Can we use a helper method here for `eloUpdate`? same as what we are doing here for instance:
https://github.com/coincoinche/coincoinche/blob/master/server/src/main/java/com/coincoinche/websockets/messages/NewStateMessage.java#L94-L112 
I think the code will be more readable and consistent.

But, if you are going to clean these JSON serializations anyway, you can ignore this comment.

Reply:
That's my next task :)